src/main/java/ch/uzh/ifi/hase/soprafs22/game/logger/GameLogger.java

```diff
@@ -0,0 +1,25 @@
+package ch.uzh.ifi.hase.soprafs22.game.logger;
+
+/**
+ * A Logger for {@link ch.uzh.ifi.hase.soprafs22.game.Game}.
+ */
+public interface GameLogger {
+
+    /**
+     * Inform that a unit was killed.
+     * @param turn in which it was killed.
+     * @param playerId id of player that lost the unit.
+     */
+    void unitKilledAtTurn(int turn, long playerId);
```

Comment:
We may also want to know which unit it was ? Where it was? What killed it? Who killed?

Reply:
I am not sure why we need to know all those things. I collected the minimum information based on the mocks in MileStone2Final.pdf, page 39. If we are targeting other kinds of statistics let me know and I will change it to cover those.

If we are not going to implement any statistics based on that information, it seems like a waste of time and effort.

Reply:
I see, valid point. If we would need to expand it in the future we might want it. But as we have our requirements we should do just that 👍